src/search/landmarks/exploration.cc

```diff
@@ -140,15 +148,16 @@ void Exploration::setup_exploration_queue(
       looping over all unary operators). Note however that this can lead to
       an overapproximation, e.g. if the effect e1 also has condition c.
     */
-    unordered_set<int> op_ids_to_mark(excluded_op_ids.begin(),
-                                      excluded_op_ids.end());
-    for (OperatorProxy op : task_proxy.get_operators()) {
-        for (EffectProxy effect : op.get_effects()) {
-            if (effect.get_conditions().empty()
-                && propositions[effect.get_fact().get_variable().get_id()]
-                [effect.get_fact().get_value()].excluded) {
-                op_ids_to_mark.insert(op.get_id());
-                break;
+    unordered_set<int> op_ids_to_mark;
+    if (!use_unary_relaxation && !excluded_props.empty()) {
```

Comment:
Do we still need the check for !excluded_props.empty()? The advantage of checking it is that we don't do the loop if we have no excluded propositions; but does this happen in practice now that we only have excluded props (i.e. no excluded ops)?